### ISSUE TITLE
Attention is all you need

### ISSUE THREAD

**Issue by chaerin314** [gdscewha-3rd/Study-PaperReview#36]
- Transformer, the ﬁrst sequence transduction model based entirely on 
attention
- Architecture
![Screenshot_20220626-162351_Polaris Office](https://user-images.githubusercontent.com/95239486/175804091-f0b8684a-caae-4248-baed-e706bcb1c2e7.jpg)
- Training
    - Training data: standard WMT 2014 English-German dataset consisting of about 4.5 million sentence pairs(byte-pair encoding)
    - Regularization
        - Residual dropout
        - Label smoothing
    - Optimizer: Adam
- Conclusion
    - WMT 2014 English-to-German, WMT 2014 English-to-French translation tasks에서 SOTA model
